### PR TITLE
[17.0][MIG] account_journal_restrict_mode: Migration to 17.0

### PR DESCRIPTION
**Pull request by LauraCForgeFlow**
Standard migration to version 17.0

### CODE REVIEW

File: account_journal_restrict_mode/models/account_journal.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import _, api, fields, models
+from odoo.exceptions import UserError
+
+
+class AccountJournal(models.Model):
+    _inherit = "account.journal"
+
+    restrict_mode_hash_table = fields.Boolean(default=True)
```
Comment (JordiMForgeFlow) [inline account_journal_restrict_mode/models/account_journal.py]:
```suggestion
    restrict_mode_hash_table = fields.Boolean(default=True, readonly=True)
```
Maybe we should also enforce readonly True so it is not modifiable in the view, although we already have a constrain

Comment (LauraCForgeFlow) [reply account_journal_restrict_mode/models/account_journal.py]: @JordiMForgeFlow Done! :)

**Review by JordiMForgeFlow — APPROVED** [review]

**Comment by JordiMForgeFlow** [comment]
@LauraCForgeFlow could you rebase? :)

**Comment by LauraCForgeFlow** [comment]
@JordiMForgeFlow Done!!

**Review by AaronHForgeFlow — APPROVED** [review]
Code review LGTM

**Pull request merged**